WeatherForecast/WeatherForecast/Model/Weather.swift

```diff
@@ -45,18 +45,26 @@ class City: Decodable {
 }
 
 // MARK: - Coord
-class Coord: Decodable {
+struct Coord: Decodable {
     let lat, lon: Double
 }
 
 // MARK: - Temperature Unit
 enum TempUnit: String {
     case metric, imperial
+    
     var expression: String {
         switch self {
         case .metric: return "℃"
         case .imperial: return "℉"
         }
     }
+    
+    var title: String {
+        switch self {
+        case .metric: return "섭씨"
+        case .imperial: return "화씨"
+        }
+    }
 }
```

Comment:
만약 섭씨와 화씨 말고 다른 유닛이 추가된다면 OCP를 지키면서 코드를 수정할 수 없을 것 같아요
TempUnit을 참조하고 있는 객체에서 switch문을 사용하고 있다면 하나씩 찾아가면서 수정을 해야 하지 않을까요? ㅎㅎ

아래와 같은 방법으로 OCP를 지키면서 확장해볼 수 있을 것 같아요
문법이 틀렸을 수도 있는데요 ㅎㅎ 참고만 부탁드려요

```suggestion
protocol TempUnit {
  var expression: String { get }
  var title: String { get }
}

struct MetricTempUnit: TempUnit {
  var expression: String {
    "℃"
  }
  
  var title: String {
    "섭씨"
  }
}
```

Reply:
리뷰를 달고보니 보너스 스텝에서 완성하셨네요! 잘 하셨습니다 ㅎㅎ
요거는 머지할게요